qbx_consumables/server.lua

```diff
@@ -32,49 +64,40 @@ end
 
 for alcohol, params in pairs(config.consumables.alcohol) do
     exports.qbx_core:CreateUseableItem(alcohol, function(source, item)
-        local player = exports.qbx_core:GetPlayer(source)
-        if not player then return end
-
         local drank = lib.callback.await('consumables:client:DrinkAlcohol', source, params.alcoholLevel, params.anim, params.prop)
         if not drank then return end
         if not exports.ox_inventory:RemoveItem(source, item.name, 1, nil, item.slot) then return end
-        local playerState = Player(source).state
 
-        local sustenance = (playerState.thirst or 0) + math.random(params.min, params.max)
+        local sustenance = math.random(params.min, params.max)
```

Comment:
Why this change? you're literally allowing an error to occur..

Reply:
check the implementation of addThirst